README.md

```diff
@@ -2,11 +2,876 @@
+
+A sample OpenTelemetry configuration using LogExporter is shown below: 
+
+```java
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import io.grpc.ManagedChannelBuilder;
+import io.opentelemetry.OpenTelemetry;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.trace.SpanProcessor;
+import io.opentelemetry.sdk.trace.export.SimpleSpanProcessor;
+import io.opentelemetry.trace.Tracer;
+import io.opentelemetry.exporters.jaeger.JaegerGrpcSpanExporter;
+import io.opentelemetry.exporters.logging.*;
+
+@Configuration
+public class OtelConfig {
+  private static tracerName = "foo"; \\TODO:
+  @Bean
+  public Tracer otelTracer() throws Exception {
+    final Tracer tracer = OpenTelemetry.getTracer(tracerName);
+
+    SpanProcessor logProcessor = SimpleSpanProcessor.newBuilder(new LoggingSpanExporter()).build();
+    OpenTelemetrySdk.getTracerProvider().addSpanProcessor(logProcessor);
+    
+    return tracer;
+  }
+}
+```
+
+
+The file above configures an OpenTelemetry tracer and a span processor. The span processor builds a log exporter which will output spans to the console. Similarly, one could add another exporter, such as the `JaegerExporter`, to visualize traces on a different back-end. Similar to how the `LoggingExporter` is configured, a Jaeger configuration can be added to the `OtelConfig` class above. 
+
+Sample configuration for a Jaeger Exporter:
+
+```java
+
+SpanProcessor jaegerProcessor = SimpleSpanProcessor
+        .newBuilder(JaegerGrpcSpanExporter.newBuilder().setServiceName(tracerName)
+            .setChannel(ManagedChannelBuilder.forAddress("localhost", 14250).usePlaintext().build())
+            .build())
+        .build();
+OpenTelemetrySdk.getTracerProvider().addSpanProcessor(jaegerProcessor);
+```
+     
+### Project Background
+
+Here we will create rest controllers for `FirstService` and `SecondService`.
+`FirstService` will send a GET request to `SecondService` to retrieve the current time. After this request is resolved, `FirstService` then will append a message to time and return a string to the client. 
+
+## Manual Instrumentation with Java SDK
+
+### Add OpenTelemetry to FirstService and SecondService
+
+Required dependencies and configurations for FirstService and SecondService projects can be found [here](#setup-for-manual-instrumentation).
+
+### FirstService
+
+1. Ensure OpenTelemetry dependencies are included
+2. Ensure an OpenTelemetry Tracer is configured
+
+3. Ensure a Spring Boot main class was created by the Spring initializer
+
+```java
+@SpringBootApplication
+public class FirstServiceApplication {
+
+  public static void main(String[] args) throws IOException {
+    SpringApplication.run(FirstServiceApplication.class, args);
+  }
+}
+```
+
+4. Create a RestController for FirstService
+5. Create a span to wrap the FirstServiceController
```

Comment:
[Question] You're wrapping the method, not the entire controller, right? Also, are you wrapping the body?

Reply:
I'm only wrapping the method here. In later examples I wrap the full controller